docs/includes/strategy-exit-comparisons.md

```diff
@@ -0,0 +1,17 @@
+## Exit logic comparisons
+
+Freqtrade allows your strategy to implement different exit logics.
```

Comment:
"logics" sounds weird in English - most would use "logic" as a collective noun, rather than a plural.